ui/pages/confirmations/components/gas-details-item/gas-details-item.test.js

```diff
@@ -32,7 +32,8 @@ const render = ({ contextProps } = {}) => {
       preferences: {
         useNativeCurrencyAsPrimaryCurrency: true,
       },
-      gasFeeEstimates: mockEstimates[GasEstimateTypes.feeMarket],
+      gasFeeEstimates:
+        mockEstimates[GasEstimateTypes.feeMarket].gasFeeEstimates,
```

Comment:
for further context, the gasFeeEstimates that should have been mocked here were nested one layer deeper than we were previously accessing